source/blog/2016-12-19-last-week-in-manageiq-armor-like-tenfold-regions-hashes.html.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Last Week in ManageIQ: Armor like Tenfold Regions Hashes"
+date: 2016-12-16
+author: drew
+tags: LWIMIQ
+comments: true
+published: true
+---
+
+Two improved pull requests for Elven-kings under the sky,
+Three fixed for Dwarf-lords in their halls of stone,
```

Comment:
If you want this verse to appear like this, you might need to do some markdown foo: it'll render this as one big paragraph